lib/manager/cocoapods/artifacts.ts

```diff
@@ -13,7 +13,7 @@ import { getRepoStatus } from '../../util/git';
 import { regEx } from '../../util/regex';
 import type { UpdateArtifact, UpdateArtifactsResult } from '../types';
 
-const pluginRegex = /^\s*plugin\s*(['"])(?<plugin>[^'"]+)\1/; // TODO #12070
+const pluginRegex = regEx(`^\\s*plugin\\s*(['"])(?<plugin>[^'"]+)(['"])`);
```

Comment:
Are you sure on this one? I'm not familiar with `\1` syntax

Reply:
It matches those lines.

```
plugin "cocoapods-acknowledgements"
```

it uses a backreference, which isn't supported by RE2
https://github.com/uhop/node-re2#backreferences

I'm not sure if we need to be so strict here, as i think the will be no package with a quote in name.

Reply:
> Are you sure on this one? I'm not familiar with `\1` syntax

Yes I am pretty sure. It's task is  to prevent us from writing the same thing repeatedly, which makes the regex long and decreases readability.